README.md

```diff
@@ -62,34 +62,39 @@ provides the following:
 
 + Partial support for firewalls:
 
-  Currently, you can only re-direct the URLs scie-pants uses to fetch [Python Build Standalone](
-  https://python-build-standalone.readthedocs.io/en/latest/) CPython distributions it uses to
-  bootstrap Pants with. This is done by exporting a `PANTS_BOOTSTRAP_URLS` environment variable
+  Currently, you can re-direct the URLs used to fetch:
+
+    + [Python Build Standalone](https://python-build-standalone.readthedocs.io/en/latest/) CPython
+      distributions used to bootstrap Pants.
+    + Pants PEX release assets which contain Pants as a single-file application.
+
+  This is done by exporting a `PANTS_BOOTSTRAP_URLS` environment variable
   specifying the path to a JSON file containing a mapping of file names to URLS to fetch them from
   under a top-level `"ptex"` key. For example:
   ```json
   {
     "ptex": {
-      "cpython-3.8.16+20230507-x86_64-unknown-linux-gnu-install_only.tar.gz": "https://github.com/indygreg/python-build-standalone/releases/download/20230507/cpython-3.8.16%2B20230507-x86_64-unknown-linux-gnu-install_only.tar.gz",
-      "cpython-3.8.16+20230507-aarch64-apple-darwin-install_only.tar.gz": "https://github.com/indygreg/python-build-standalone/releases/download/20230507/cpython-3.8.16%2B20230507-aarch64-apple-darwin-install_only.tar.gz",
-      "cpython-3.9.16+20230507-x86_64-unknown-linux-gnu-install_only.tar.gz": "https://github.com/indygreg/python-build-standalone/releases/download/20230507/cpython-3.9.16%2B20230507-x86_64-unknown-linux-gnu-install_only.tar.gz",
-      "cpython-3.9.16+20230507-aarch64-apple-darwin-install_only.tar.gz": "https://github.com/indygreg/python-build-standalone/releases/download/20230507/cpython-3.9.16%2B20230507-aarch64-apple-darwin-install_only.tar.gz",
+      "cpython-3.8.16+20230507-x86_64-unknown-linux-gnu-install_only.tar.gz": "https://example.com/cpython-3.8.16%2B20230507-x86_64-unknown-linux-gnu-install_only.tar.gz",
+      "cpython-3.8.16+20230507-aarch64-apple-darwin-install_only.tar.gz": "https://example.com/cpython-3.8.16%2B20230507-aarch64-apple-darwin-install_only.tar.gz",
+      "cpython-3.9.16+20230507-x86_64-unknown-linux-gnu-install_only.tar.gz": "https://example.com/cpython-3.9.16%2B20230507-x86_64-unknown-linux-gnu-install_only.tar.gz",
+      "cpython-3.9.16+20230507-aarch64-apple-darwin-install_only.tar.gz": "https://example.com/cpython-3.9.16%2B20230507-aarch64-apple-darwin-install_only.tar.gz",
+      "pants.2.18.0-cp9-linux-x86_64.pex": "https://example.com/pants.2.18.0-cp9-linux-x86_64.pex",
+      ...
     }
   }
   ```
-  To see the current mapping used by your version of `scie-pants` you can run:
+  To see the current mapping used by your version of `scie-pants` for Python Build Standalone you
+  can run:
   ```
   $ SCIE=inspect scie-pants | jq .ptex
   ```
   You'll need to run this once for each platform you use `scie-pants` on to gather all mappings
   you'll need; e.g.: once for Linux x86_64 and once for Mac ARM.
```

Comment:
I think this suggestion won't reveal the pants pex names or the original/canonical URLs the user needs to download and supply. Is there some way we can hook that up or some alternative we can offer? Working it out manually seems like hard work...

Reply:
To this point and the next, yeah I struggled to edit the wording to be inclusive of all the aspects of this new JSON file being partially for the `scie-pants` bootstrapping and partially for Pants PEX downloading.

So I'm open to suggestions, but this was my result.

---

Since the `SCIE=inspect` comes from the upstream `scie` tools, so I don't expect to find a way to change that. I'm expecting that if someone needs this, they'll hit some kind of error in their environment and be able to work backwards from there.

Reply:
OK I spent a little time breaking it down father in this section